source/userguide/custom_action.rst

```diff
@@ -0,0 +1,219 @@
+Working with Custom Action
+==========================
+
+MindMeld 4.3 introduces Custom Actions which allow MindMeld developers to further separate their business
+logic from the conversational application. This paradigm shift helps to reduce the barriers to build and
```

Comment:
I dont think we need to mention this as a paradigm shift.

Reply:
Same, I dont think paradigm shift is needed here.

Reply:
Done